languages/python/exercises/concept/conditionals/.docs/introduction.md

```diff
@@ -0,0 +1,44 @@
+## Conditionals
+
+Conditionals are the most common statements used in programming. Conditionals determine the flow of control in a program. In python the `if` statement is used to control the flow of execution. An important aspect of conditions in an `if` statment is that it will be resolved into `boolean` type to determine the flow of execution.
+
+The basic syntax of an `if` statement is as follows
+
+```python
+x = 10
+y = 5
+# Variant 1
+if x > y:
+    print("x is greater than y")
+
+>>> x is greater than y
+
+# Variant 2
+x = 5
+y = 10
+
+if x > y:
+    print("x is greater than y")
+else:
+    print("y is greater than x")
+
+>>> y is greater than x
+
+# Variant 3
+x = 5
+y = 10
+z = 20
+if x > y:
+    print("x is greater than y and z")
+elif y > z:
+    print("y is greater than x and z")
+else:
+    print("z is great than x and y")
+>>> z is great than x and y
+```
```

Comment:
There is quite a bit of code here. Maybe we can shorten this somehow? One option would be to do the following:

````suggestion
```python
# Variant 1
if 2 > 7:
    print("2 is greater than 7")
elif 2 > 3:
    print("2 is greater than 3")
else:
    print("2 is less than or equal to 2")
    
>>> 2 is less than or equal to 2
```
````

You could then add a textual remark stating that the `elif` and `else` branches are optional. What do you think?